deploy/02_deploy_vending_machine.ts

```diff
@@ -0,0 +1,29 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { execute } = deployments
+
+  const KeepToken = await deployments.get("KeepToken")
+  const T = await deployments.get("T")
+
+  const vendingMachine = await deployments.deploy("VendingMachine", {
+    from: deployer,
+    args: [KeepToken.address, T.address, 1000000000, 10000000000],
```

Comment:
`_wrappedTokenAllocation` should be set to `KEEP.totalSupply()`. Ideally, we should call a function on the resolved `KeepToken` instance because this number will be a little different on testnet and mainnet because of slashing.

Reply:
See d29e4fc5e149e17064ce5cd74a1667c79b7989f9.